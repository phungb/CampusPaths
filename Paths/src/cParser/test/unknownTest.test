# Test Unknown Buildings
r
FOO
BAR

# terminate the program
q