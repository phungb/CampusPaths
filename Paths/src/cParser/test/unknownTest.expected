Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Test Unknown Buildings
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building FOO
Unknown building BAR

Enter an option ('m' to see the menu):
# terminate the program